sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/InputFile.java

```diff
@@ -41,7 +41,7 @@
  *
  * @since 4.2
  */
-public interface InputFile extends InputPath {
+public interface InputFile extends IndexedFile, InputPath {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) "InputPath" is already extended by "IndexedFile"; there is no need to implement it here. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1939)